woocommerce-gateway-ebanx/templates/instalments.php

```diff
@@ -1,5 +1,5 @@
 <!-- Installments -->
-<?php if ($max_instalments > 1) : ?>
+<?php if ( count($instalments_terms) > 1 ) : ?>
```

Comment:
This is great, we got rid of the static loop in the template and now we can count stuff =D